pxr/imaging/hd/strategyBase.h

```diff
@@ -72,7 +72,8 @@ class HdAggregationStrategy {
     virtual HdBufferSpecVector GetBufferSpecs(
         HdBufferArraySharedPtr const &bufferArray) const = 0;
 
-    /// Returns the accumulated GPU resource allocation      /// for items in the BufferArray passed as parameter
```

Comment:
Discovered (by accident) that this comment line has a weird character at the end.